extensions/community_extensions/Protected Audience Support.md

```diff
@@ -0,0 +1,125 @@
+# Bid Request Values
+
+## Object: InterestGroupAuctionSupport 
+
+This object allows sellers to signal interest group auction support for an Imp. 
+
+<table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>ae</code></td>
+    <td>integer;<br>default 1</td>
+    <td>Interest Group auction environment support for this impression: <br>
```

Comment:
This may change depending on the output of https://github.com/InteractiveAdvertisingBureau/openrtb2.x/pull/89

Reply:
it appears i should just close 89 and those changes should be incorporated here